Classes/ViewHelpers/Outlet/ArgumentViewHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace FluidTYPO3\Flux\ViewHelpers\Outlet;
+
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+use FluidTYPO3\Flux\Outlet\OutletArgument;
+use FluidTYPO3\Flux\ViewHelpers\AbstractFormViewHelper;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+use TYPO3\CMS\Fluid\Core\Rendering\RenderingContextInterface;
+
+/**
```

Comment:
This should have at least a minimal description since it gets rendered in VH docs.